pages/_app.js

```diff
@@ -6,6 +6,9 @@ export default function App({ Component, pageProps }) {
   const allZukunftsanalyseEntries = placeholderEntries.filter(
```

Comment:
```suggestion
  const allFutureAnalysisEntries = placeholderEntries.filter(
```
You should write all of your code (variable names included) in english. Only the content should be written in German, when that's your app's language.